applications/main/momentum_app/scenes/momentum_app_scene_misc_vgm.c

```diff
@@ -31,58 +38,73 @@ static void momentum_app_scene_misc_vgm_colors_changed(VariableItem* item) {
         variable_item_list_get(app->var_item_list, VarItemListIndexBackground),
         index != VgmColorModeCustom,
         NULL);
-    expansion_disable(app->expansion);
-    expansion_enable(app->expansion);
 }
 
 static const struct {
     char* name;
-    Rgb565Color color;
+    RgbColor color;
 } vgm_colors[] = {
     // clang-format off
-    {"Orange", {0xFC00}},
-    {"Black", {0x0000}},
-    {"Red", {0xF800}},
-    {"Maroon", {0x8000}},
-    {"Yellow", {0xFFE0}},
-    {"Olive", {0x8400}},
-    {"Lime", {0x07E0}},
-    {"Green", {0x0400}},
-    {"Aqua", {0x07EF}},
-    {"Cyan", {0x069A}},
-    {"Azure", {0x03FF}},
-    {"Teal", {0x0410}},
-    {"Blue", {0x001F}},
-    {"Navy", {0x0010}},
-    {"Purple", {0x8010}},
-    {"Fuchsia", {0xF81F}},
-    {"Pink", {0xA8F5}},
-    {"Brown", {0xA145}},
-    {"White", {0xFFFF}},
+    {"Off", {{0, 0, 0}}},
+    {"Rainbow", {{0, 0, 0}}},
+    {"RgbMod", {{0, 0, 0}}},
```

Comment:
is the intent to set rainbow and rgbmod modes with the foreground and background toggles? if so, we can get rid of the other entry in the menu for "VGM Colors"

Reply:
Yes because you can now set the background or the foreground with rainbow and not one of them.
Wanted to ask if we should restructure or keep it like how I did it.